apps/showcase/src/app/home/home.template.html

```diff
@@ -1,6 +1,41 @@
 <div class="d-flex flex-column gap-2 align-items-center justify-center flex-fill">
-  <h1>Otter</h1>
-  <p>This is a showcase application.</p>
-  <p>You can find examples of Otter features with some explanations.</p>
-  <img width="500" [src]="'otter-a.svg' | dynamicContent" class="otter-logo">
+  <img width="350" [src]="'otter-a.svg' | dynamicContent" class="otter-logo" alt="Otter">
+  <h1 class="h1 display-1 text-center mt-0 mb-5"><b>Build easily a customizable Angular based application</b></h1>
+  <h2 class="h4 mt-5 pt-5 text-secondary">
+    The Otter project is a highly modular framework whose goal is to provide a common platform to <b>accelerate</b> and <b>facilitate</b> the development of <span class="text-danger">Angular</span> based Web Applications.<br />
+    The framework offer the capability to customize the application using <b>Content Management System</b>.
+  </h2>
 </div>
+
+<div class="get-start-section d-flex flex-column gap-2 align-items-center justify-center flex-fill">
+  <h3 class="display-3 text-center mt-0 mb-5">Get started with Otter</h3>
+  <p class="text-secondary">
+    To start a new project you just need to follow the guide <a target="_blank" href="https://github.com/AmadeusITGroup/otter/blob/main/docs/core/START_NEW_APPLICATION.md">on this page</a> or use the following command lines:
```

Comment:
the link is not very meaningful out of context
maybe something like
```suggestion
    To start a new project you just need to follow the <a target="_blank" href="https://github.com/AmadeusITGroup/otter/blob/main/docs/core/START_NEW_APPLICATION.md">start new application guide</a> or use the following command lines:
```

Reply:
A bit too many "start" in the short sentence

Reply:
```suggestion
    To start a new project you just need to follow the <a target="_blank" href="https://github.com/AmadeusITGroup/otter/blob/main/docs/core/START_NEW_APPLICATION.md">new application guide</a> or use the following command lines:
```
:D